data/minio/bucket/custom_files/model_metadata.json

```diff
@@ -2,43 +2,43 @@
     "action_space": [
         {
             "steering_angle": -30,
-            "speed": 1.0,
+            "speed": 1.5,
```

Comment:
1.5 is very fast for a 30 degree corner, we should provide a default action space that is definitely slow enough to actually learn to complete a lap in reasonable time.

Reply:
New speeds are set.